pyuvdata/utils.py

```diff
@@ -118,7 +118,9 @@
                     "ee": ["e", "e"], "nn": ["n", "n"],
                     "en": ["e", "n"], "ne": ["n", "e"],
                     "rr": ["r", "r"], "ll": ["l", "l"],
-                    "rl": ["r", "l"], "lr": ["l", "r"]}
+                    "rl": ["r", "l"], "lr": ["l", "r"],
+                    "pI": ["I", "I"], "pQ": ["Q", "Q"],
+                    "pU": ["U", "U"], "pV": ["V", "V"]}
```

Comment:
This confuses me. This is used to map calibration solutions (which are per antpol) to visibilities which have two antpols. I don't understand how you can have a stokes parameter feed.

Reply:
There's a longer story behind this, but this dict is also what's used for writing out the CASA `FEEDS` table, which I think is also used to map baselines back to individual receivers/inputs for the sake of calibration (among other things). I added it because I had an issue reading in a UVFITS file with a polarization code of 1 (Stokes I/pseudo-I) and writing that back out in CASA. Given that we have the hooks for supporting read in of the former, I felt that adding support for writing out the latter was merited. It could be that we need a separate dictionary for storing such information, or that we don't want to support such "pseudo-Stokes" parameters out of various objects -- I'm certainly amenable to other ideas (or dropping it all together), especially given that this isn't some major issue, just a thing accidentally stumbled upon in testing.

Reply:
Maybe we can just add a comment for the future reader that this dict is also used in measurement sets and the pseudo-stokes lines are needed to be able to write measurement sets for pseudo-stokes data.

Reply:
Done! Updated in b8efacc9eb638a229d745fb4bb9addd6248c2879